src/Snippets/Docs/Chaos.Result.cs

```diff
@@ -0,0 +1,83 @@
+using System.Net;
+using Polly.Retry;
+using Polly.Simmy;
+using Polly.Simmy.Outcomes;
+
+namespace Snippets.Docs;
+
+internal static partial class Chaos
+{
+    public static void ResultUsage()
+    {
+        #region chaos-result-usage
+        // Outcome using the default options.
+        // See https://www.pollydocs.org/chaos/result#defaults for defaults.
+        var optionsDefault = new OutcomeStrategyOptions<HttpStatusCode>();
```

Comment:
```suggestion
        var optionsDefault = new OutcomeStrategyOptions<HttpResponseMessage>();
```

^^ feels more natural to how this will be used in HTTP scenarios

Reply:
This would require to rewrite the entire example to use the `HttpResponseMessage` everywhere. Are you okay with that? @martintmk

Reply:
Sure, I see no problem with that.

Reply:
Done